.github/workflows/push-staging.yml

```diff
@@ -0,0 +1,25 @@
+name: 1 | Push changes to releases/staging
+
+# This CI job is responsible for pushing the current contents of the `main` branch to the
+# releases/staging branch, which will in turn kick off a deploy to the staging environment.
```

Comment:
Will the deploy me defined in a separate action?

Reply:
Yes!